# **Big Data: Challenges and Strategies in Data Generation, Storage, and Retrieval**

In [47]:
import pandas as pd

results = pd.read_csv('C:\\Users\\yeiso\\Documents\\GitHub\\Special-Topics-in-Data-Analytics-CSIS-4260-002\\docs\\results.csv')


In [48]:
print(results.info())


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6 entries, 0 to 5
Data columns (total 8 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   Order_ID      6 non-null      int64  
 1   Date_Created  6 non-null      object 
 2   Category      6 non-null      object 
 3   City          6 non-null      object 
 4   Price         6 non-null      float64
 5   Product_Name  6 non-null      object 
 6   State         6 non-null      object 
 7   Sub_Category  6 non-null      object 
dtypes: float64(1), int64(1), object(6)
memory usage: 516.0+ bytes
None


In [49]:
print(results.head())


   Order_ID Date_Created                     Category               City  \
0         3   2019-06-12  "Office Supplies, Supplies"        Los Angeles   
1         2   2019-11-08                    Furniture          Henderson   
2         4   2020-04-15              Office Supplies            Concord   
3         6   2019-12-09              Office Supplies  "Fremont, Quebec"   
4         1   2019-11-08                    Furniture          Henderson   

    Price                                       Product_Name           State  \
0   14.62  Self-Adhesive Address Labels for Typewriters b...      California   
1  731.94  "Hon Deluxe Fabric Upholstered Stacking Chairs...        Kentucky   
2   45.90                                         Xerox 1967  North Carolina   
3   61.38      "Acco Six-Outlet Power Strip, 4' Cord Length"        Nebraska   
4  189.99                  Bush Somerset Collection Bookcase        Kentucky   

                Sub_Category  
0                     Labels  


In [51]:
print(results.columns)


Index(['Order_ID', 'Date_Created', 'Category', 'City', 'Price', 'Product_Name',
       'State', 'Sub_Category'],
      dtype='object')


In [50]:
print(results.shape)


(6, 8)


In [52]:
print(results.dtypes)

Order_ID          int64
Date_Created     object
Category         object
City             object
Price           float64
Product_Name     object
State            object
Sub_Category     object
dtype: object


| Order_ID | Date_Created | Category                  | City             | Price  | Product_Name                                           | State         | Sub_Category            |
|----------|--------------|---------------------------|------------------|--------|--------------------------------------------------------|---------------|-------------------------|
| 3        | 2019-06-12   | Office Supplies, Supplies | Los Angeles      | 14.62  | Self-Adhesive Address Labels for Typewriters by Universal | California    | Labels                  |
| 2        | 2019-11-08   | Furniture                 | Henderson        | 731.94 | Hon Deluxe Fabric Upholstered Stacking Chairs, Rounded Back| Kentucky      | Chairs, sits            |
| 4        | 2020-04-15   | Office Supplies            | Concord          | 45.9   | Xerox 1967                                             | North Carolina | Paper                  |
| 6        | 2019-12-09   | Office Supplies           | Fremont, Quebec   | 61.38  | Acco Six-Outlet Power Strip, 4' Cord Length             | Nebraska      | Appliances              |
| 1        | 2019-11-08   | Furniture                 | Henderson        | 189.99 | Bush Somerset Collection Bookcase                      | Kentucky      | Bookcases, miscellaneous |
| 5        | 2019-12-05   | Office Supplies            | Seattle, New York | 907.15 | Fellowes PB200 Plastic Comb Binding Machine            | Washington    | Binders                 |
